code/game/machinery/recycler.dm

```diff
@@ -15,14 +15,17 @@ var/const/SAFETY_COOLDOWN = 100
 	var/eat_dir = WEST
 	var/amount_produced = 1
 	var/datum/material_container/materials
+	var/circuit = /obj/item/weapon/circuitboard/recycler
+	var/transmute = FALSE
 
 /obj/machinery/recycler/New()
 	// On us
 	..()
 	component_parts = list()
-	component_parts += new /obj/item/weapon/circuitboard/recycler(null)
-	component_parts += new /obj/item/weapon/stock_parts/matter_bin(null)
-	component_parts += new /obj/item/weapon/stock_parts/manipulator(null)
+	component_parts += new circuit(null)
+	if(!transmute)
+		component_parts += new /obj/item/weapon/stock_parts/matter_bin(null)
+		component_parts += new /obj/item/weapon/stock_parts/manipulator(null)

```

Comment:
Doesn't the machine new need to be called? I did this because I wasn't sure how to make it skip a level of new and call the parent of the parent. Or does that not matter in this case?